src/components/Pagination.jsx

```diff
@@ -51,6 +78,12 @@ function Pagination({
       {paginationRange.map((pageNumber) => {
         const key = nanoid();
 
+        // highlight the current page
+        let highlight = "false";
+        if (pageNumber === currentPage) {
+          highlight = "page";
+        }
```

Comment:
It is also cleaner to use the candition for disabling the button directly as the value of the "disabled" attribute instead of creating a variable and reassigning it.